static-site/src/components/ListResources/IndividualResource.tsx

```diff
@@ -158,12 +158,14 @@ export const IndividualResource = ({resource, isMobile}: IndividualResourceProps
     )
   }
 
+  const description = resource.nVersions ? `Last known update on ${resource.lastUpdated}` : "";
```

Comment:
Suggestion: condition on `resource.lastUpdated` directly

```suggestion
  const description = resource.lastUpdated ? `Last known update on ${resource.lastUpdated}` : "";
```

Reply:
I want to say I tried this and it didn't DTRT …but I don't remember what it did. Will give it a shot.

Reply:
Nope, this works great, changed.